### PR TITLE
Support multiple accounts in VM mode

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by chriseth** [comment]
Please rebase.

**Comment by axic** [comment]
That was a lot of changes, this should work.

**Comment by chriseth** [comment]
Hm, would be good to have a button to create a new account (which could work both in simulation and in "attached to node" mode). Ok if you want to do it in another pr.

**Comment by axic** [comment]
Re: creating new accounts is not part of the `eth_` RPC branch. There is a new one with `personal_` prefix which I am not sure how much of it is exported via RPC.

I can certainly add the code to generate a new account in the VM, but not sure about the RPC part.  I would like someone else to do the UI part too :)

**Comment by chriseth** [comment]
argh, we need a ui person :)
as far as I know, `personal_` is now also just part of `web3.js`. If not, we can just do it for the virtual node, where it is more important anyway.

**Comment by axic** [comment]
And using `personal_` would mean replacing all the other calls for `eth_accounts`, because as far as I understand it is a separate keystore?

This means it probably makes sense merging this PR if you like it, and working on any of those separately.

File: assets/js/universal-dapp.js [hunk 1]
```diff
@@ -25,6 +23,26 @@ function UniversalDApp (contracts, options) {
     }
 
 }
+
+UniversalDApp.prototype.addAccount = function (privateKey, balance) {
+  if (this.accounts) {
```
Comment (chriseth) [inline assets/js/universal-dapp.js:28]: Indentation is different than in the rest of the file. Please fix with next PR.

**Pull request merged**